app/models/vim_performance_daily.rb

```diff
@@ -25,7 +25,7 @@ def self.excluded_cols_for_expressions
   def self.find(cnt, *args)
     raise "Unsupported finder value #{cnt}" unless cnt == :all
 
-    Vmdb::Deprecation.deprecation_warning(:find, :find_entries)
+    Vmdb::Deprecation.deprecation_warning(:find, :find_entries, caller(1))
```

Comment:
@matthewd Can we review/merge this for now?  Any changes to rails to fix this will still accept a subset of caller/caller_locations.  I'm not sure when I'll get to moving things around in Rails to open a PR to fix it.